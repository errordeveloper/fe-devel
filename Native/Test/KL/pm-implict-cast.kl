/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

function testRValue( Boolean boolean ) {
  report("testRValue: " + boolean);
}

operator entry() {
  testRValue(42);
  testRValue(0);

  Scalar one = 1.0, zero = 0.0;
  testRValue(one);
  testRValue(zero);
}
