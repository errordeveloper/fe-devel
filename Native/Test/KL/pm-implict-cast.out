testRValue: true
testRValue: false
testRValue: true
testRValue: false
